configs/platforms/debian-9-amd64.rb

```diff
@@ -1,7 +1,7 @@
 platform 'debian-9-amd64' do |plat|
-  plat.servicedir '/lib/systemd/system'
   plat.defaultdir '/etc/default'
-  plat.servicetype 'systemd'
+  plat.servicetype 'systemd', servicedir: '/lib/systemd/system'
+  plat.servicetype 'sysv', servicedir: '/etc/init.d'
```

Comment:
This might not be needed as pxp-agent-vanagon should just provide to files for service configuration and puppet-agent#main will be responsible of configuring correctly each platform

Reply:
I think service definition is used on https://github.com/puppetlabs/pxp-agent-vanagon/blob/0dbce55b57b4c8198ed73f4882d85e036fb9604c/configs/components/pxp-agent.rb#L99-L100 to select which files to install

Reply:
Yes, makes sense now